### DISCUSSION TITLE
Initializing connection results in UAC

### DISCUSSION THREAD

**Discussion by Averageunknownuser** [rustdesk/rustdesk#7191] [General]
### Bug Description

Hello everyone, 

after initializing a connection, my partner receives an UAC before getting the option to accept the connection. 

### How to Reproduce

I'm starting a connection request, fill in the ID Number and hit connect
My partner receives a UAC message.
Meanwhile on my display i get the chance to insert the password.
My partner closes the UAC and the normal support procedure continues.

### Expected Behavior

Starting a connection shouldn't need elevated rights, so the UAC shouldn't pop up on the users' display.
My partner has no admin rights and the procedure doesn't need the administrator.

### Operating system(s) on local side and remote side

Windows 11 -> Windows 10

### RustDesk Version(s) on local side and remote side

1.1.9 -> 1.2.0

### Screenshots

![Screenshot 2024-02-19 124016](https://github.com/rustdesk/rustdesk/assets/160489140/ce385b50-015f-4672-ac14-ed3712789558)

### Additional Context

i could reproduce the issue on 3 out of 3 computers. 
All of them using Windows 10, 2 out of 3 are normal laptops, the other one is a virtual machine.

**Comment by rustdesk** [comment]
We know this. That's because it attempts p2p first. It needs to create a tcp listen port for p2p connection which will trigger the UAC, you do not need to accept the UAC, though it will lead to a relay connection. For the future quick support version, we can avoid this by no p2p attempt. But, appologize, for the time being, we won't do any change for your support use case.
